code/network_analysis.py

```diff
@@ -287,3 +313,32 @@ def group_z_values(standard_group_source_prefix, cond_filepath_prefix, dist_from
   output_filename = os.path.join(os.path.dirname(__file__), "..", "results")
 
   generate_connectivity_results(z_values_store, output_filename)
+
+##perform permutation test
+# target r-values into list
+con_dmn_cer = np.ravel(z_values_store["003"]["con"]["Default-Cerebellar"]).tolist()
```

Comment:
note that these lines should be indented so that they fall into the if statement at line 298. But I'll go ahead and merge this and I'll fix this issue along with the issue of test failing and file path.